packages/neuron-ui/src/types/App/index.d.ts

```diff
@@ -281,6 +281,29 @@ declare namespace State {
     errorMsg: string
   }
 
+  enum SendType {
```

Comment:
After `SendType` define here, then we can remove https://github.com/nervosnetwork/neuron/blob/develop/packages/neuron-ui/src/components/SUDTSend/hooks.ts#L8-L15. And use `State.SendType` to replace `SendType`.

Reply:
`enum` is not a pure type, `SendType` cannot be used directly through `State.SendType`

Reply:
Gotcha

Reply:
Gotcha